docs/assets/buildwarningbanner.js

```diff
@@ -5,11 +5,13 @@
 
 /* globals window */
 
-// Display a warning banner when browsing nightly documentation build. Source parameter for maintenance of 404 redirs.
+// Display a warning banner when browsing nightly documentation build or legacy guides. Source parameter for maintenance of 404 redirs.
 if ( window.location.host === 'ckeditor5.github.io' ) {
 	const stableUrl = window.location.href.replace( 'https://ckeditor5.github.io/docs/nightly', 'https://ckeditor.com/docs' );
 
 	window.umberto.showWarningBanner(
 		`Nightly documentation ahead. Switch to the <a href="${ stableUrl }?source=nightly">stable editor documentation</a>.`
 	);
+} else if ( window.location.href.includes( '/legacy/' ) ) {
+	window.umberto.showWarningBanner( 'You are in the legacy documentation.' );
```

Comment:
"You are reading the legacy documentation"?

Reply:
<img width="1033" alt="Screenshot 2024-07-08 at 10 52 50" src="https://github.com/ckeditor/ckeditor5/assets/26603574/0fd487ec-841b-4142-9f72-2ac3fdb94554">

It looks better, tbh. I'll change it.